packages/aragon-wrapper/src/index.js

```diff
@@ -419,12 +419,26 @@ export default class Aragon {
     }
   }
 
+  /**
+   * Set the available accounts for the current user.
+   *
+   * @param {Array<string>} accounts
+   * @return {void}
+   */
+  setAccounts (accounts) {
```

Comment:
A little bit confused about the type `accounts` should be; `getAccounts()` apparently returns a `Promise` but `setAccounts()` takes in an array?

Reply:
`getAccounts` returns a promise because it falls back to calling via Web3, which returns a promise.

I will move `accounts` as an optional parameter to `init`, but `getAccounts` will always be a promise.